### DISCUSSION TITLE
Page name should be before Site name in `<title>`

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#5341] [Ideas & Feature Requests]
- Laravel Version: 9.52.0
- Nova Version: 4.22.0
- PHP Version: 8.1
- Database Driver & Version: mysql
- Operating System and Version: macOS 13.2
- Browser type and version: Chrome 109

### Description:
Hi, 
Currently the site name is first, which mean when you have multiple Nova Tabs open for the same site, you have no way of differentiating what page the tab has loaded. The title parts should be the other way around so the page name if before the site name. This is also standard SEO practice, even through the Nova admin would not be index by Search engines.

Normal tab:
![Capture d’écran, le 2023-02-20 à 10 52 57](https://user-images.githubusercontent.com/17392251/220152495-6c1e5eb0-32d5-4451-b0e2-ecfb35ee931e.png)

Condensed tab:
![Capture d’écran, le 2023-02-20 à 10 53 06](https://user-images.githubusercontent.com/17392251/220152033-4c34ec01-1e87-44ca-83c6-9b0de53a4fbd.png)

Proposal:
![Capture d’écran, le 2023-02-20 à 10 54 45](https://user-images.githubusercontent.com/17392251/220152358-3f5e0b7e-d136-4ab0-b30f-5761fe021f5e.png)

Also we are looking to add a favicon, I found the template `meta.blade.php` to override, but it's not documented anywhere.

**Comment by jsheffers** [comment]
I would love to see this as well.  We have the same problem. Looks like this is setup in `createInertiaApp()` with no way to override it.

**Comment by davidhemphill** [comment]
This has been added and will be included in the next release. 👍

**Reply by jsheffers** [reply]
Thanks David!
